pkg/distributor/ha_tracker.go

```diff
@@ -149,6 +156,23 @@ func newHATracker(cfg HATrackerConfig, limits haTrackerLimits, reg prometheus.Re
 			Name: "cortex_ha_tracker_kv_store_cas_total",
 			Help: "The total number of CAS calls to the KV store for a user ID/cluster.",
 		}, []string{"user", "cluster"}),
+
+		cleanupRuns: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_ha_tracker_elected_replicas_cleanup_cycles_total",
```

Comment:
I think it's the first time we use the "cycle" terminology. A `..._cleanup_started_total` would better fit our metrics naming.